gammapy/spectrum/energy.py

```diff
@@ -0,0 +1,293 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import print_function, division
+from astropy.units import Quantity
+from astropy.io import fits
+from ..utils.array import array_stats_str
+
+import astropy.units as u
+import numpy as np
+
+__all__ = ['Energy',
+           'EnergyBounds']
+
+class Energy(u.Quantity):
+
+    """Energy bin centers.
+
+    Stored as "ENERGIES" FITS table extensions.
```

Comment:
I'd put as docstring something a bit more generic, e.g.
```python
"""Energy quantity scalar or array.

This is a `~astropy.units.Quantity` sub-class that adds convenience methods
to handle common tasks for energy bin center arrays, like FITS I/O or generating
equal-log-spaced grids of energies.

See :ref:`gammapy-energy-handling` for further information.
```
where the ref points to a label in front of the title in the high-level docs.